doc/guides/writing-tests.md

```diff
@@ -120,7 +120,7 @@ Avoid timers unless the test is specifically testing timers. There are multiple
 reasons for this. Mainly, they are a source of flakiness. For a thorough
 explanation go [here](https://github.com/nodejs/testing/issues/27).
 
-In the event a test needs a timer, consider using the
+In the event of a ['flaky tests' situation](https://github.com/nodejs/node/wiki/Flaky-tests), consider using the
```

Comment:
@mhdawson, thanks for reply! Probably i haven't used term 'flaky tests' correctly here (i'm not native speaker :( ). I meant the same thing as you: we should use common.platformTimeout() only in situations when we should use timers and tests _are fails_ on slow platforms (like you said: arm).

You can help me, if you suggest more correct term for this sentence :)